tests/test_utils/tests.py

```diff
@@ -734,6 +734,11 @@ def test_html_contain(self):
         dom1 = parse_html('<div><p>foo</p><p>bar</p></div>')
         self.assertIn(dom2, dom1)
 
+    def test_assertInHTML(self):
```

Comment:
Minor nit: Please don't mix `snake_case` and `camelCase` in one identifier if possible. `test_html_contain_without_parents` maybe?

(This will probably come up when committers add their review.)

Reply:
I don't like doing that either but I thought I would have to since I'm unit testing the function `assertInHTML()`. Thoughts?

Reply:
Yes, that may be a good reason. Let's wait and see.